### DISCUSSION TITLE
回溯性激励发放

### DISCUSSION THREAD

**Discussion by Nami-upchain** [lbc-team/UpchainDAO#13] [Announcements]
2024年3月，GCC 通过了为登链社区捐赠的提案，[GCC 捐赠登链社区5万美金](https://learnblockchain.cn/article/7675)
此次捐赠中的一部分资金会作为过往社区主要贡献者的回溯性奖励(统计时间2024年3月15日)，请曾经在社区网站的文章、问答贡献过的、曾在DeCert上获取超过20个SBT的、曾在Gitcoin为登链社区捐赠过的、曾在社区建设中付出过其他卓越贡献的各位小伙伴们，在登链社区的奖励领取页面上线后，查看并领取奖励

**Comment by xilibi2003** [comment]
详细领取规则已经确认，前往  https://learnblockchain.cn/article/7818  查看。
